diesel-sentry/src/lib.rs

```diff
@@ -139,52 +184,6 @@ where
         result
     }
 
-    #[tracing::instrument(
-        fields(
-            db.name=%self.info.current_database,
-            db.system="postgresql",
-            db.version=%self.info.version,
-            db.statement=%query,
-            otel.kind="client",
-        ),
-        skip(self),
-    )]
-    fn execute(&mut self, query: &str) -> QueryResult<usize> {
-        let mut txn = start_sentry_db_transaction("sql.query", query);
-
-        let result = self.inner.execute(query);
-        txn.finish();
-        result
-    }
-
-    #[tracing::instrument(
-        fields(
-            db.name=%self.info.current_database,
-            db.system="postgresql",
-            db.version=%self.info.version,
-            db.statement=tracing::field::Empty,
-            otel.kind="client",
-        ),
-        skip(self, source),
-    )]
```

Comment:
I couldn't find any references to this function so I removed it since it is no longer part of connection. Not sure where it was supposed to be.